lib/src/util/csv_parser.dart

```diff
@@ -0,0 +1,78 @@
+/// CSVParser parses a stream of lines into a list of vals compliant with
+/// the CSV standard
+///
+/// See RFC4180 for details on CSV standard
+class CsvParser {
+
+  /// Takes a single line and parses it into a list of values according to the
+  /// csv standard (see RFC4180)
+  static List<String> parseLine(
+      String line) {
+    final records = <String>[];
+    var i = 0;
+    StringBuffer record;
+    while (i < line.length) {
+      record = StringBuffer();
+      if (line[i] == "\"") {
```

Comment:
Using `"` for strings makes all the code a lot harder to read (all the double quotes within strings have to be escaped), consider switching the entire DF codebase to using `'` for strings.

Reply:
Ok for switching to single quotes

Reply:
Created #7 can you review and merge that one first so I can rebase #5 and #6?